azure-devops/projects/pagopa-projects/tlscert-api-config-dev-platform-pagopa-it.tf

```
@@ -0,0 +1,74 @@
+variable "tlscert-api-config-dev-platform-pagopa-it" {
```

```suggestion
variable "tlscert-dev-api-config-dev-platform-pagopa-it" {
```
missing `dev-` prefix in vars and filename